cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -739,13 +739,15 @@ def _two_qubit_clifford_matrices(
 
 
 def _random_single_q_clifford(
-    qubit: 'cirq.Qid', num_cfds: int, cfds: Sequence[Sequence['cirq.Gate']]
+    qubit: 'cirq.Qid', num_cfds: int, cfds: Sequence[Sequence['cirq.ops.SingleQubitCliffordGate']]
 ) -> List['cirq.Operation']:
     clifford_group_size = 24
-    operations = [[gate(qubit) for gate in gates] for gates in cfds]
+    operations = [[gate.to_phased_xz_gate()(qubit) for gate in gates] for gates in cfds]
     gate_ids = list(np.random.choice(clifford_group_size, num_cfds))
     adjoint = _reduce_gate_seq([gate for gate_id in gate_ids for gate in cfds[gate_id]]) ** -1
-    return [op for gate_id in gate_ids for op in operations[gate_id]] + [adjoint(qubit)]
+    return [op for gate_id in gate_ids for op in operations[gate_id]] + [
```

Comment:
I'm surprised that this didn't cause any unit tests to fail.

Can you add a test that validates the generated circuit(s) against a known gateset that resembles what we've already published via the Virtual Quantum Engine? e.g., you might check against a [ValidatingTestDevice](https://quantumai.google/reference/python/cirq/testing/ValidatingTestDevice).

Reply:
that's because none of the test examin the internals of the circuits but treat the circuits as blackboxes with a specific behaviour => it checks things like pauli errors and other characterisitics.

added a test for now